tf-modules/s3-credentials/variables.tf

```diff
@@ -0,0 +1,79 @@
+# Required
+
+variable "prefix" {
+  type        = string
+  description = "Resource prefix unique to this deployment"
+}
+
+variable "sts_credentials_lambda_function_arn" {
+  type    = string
+}
+
+variable "tea_api_gateway_stage" {
```

Comment:
I was somewhat inclined to rename these `tea_*` variables to remove the `tea_` prefix, since in theory you could pass in values corresponding to **any** API gateway to attach the /s3credentials endpoint. And we just happen to be attaching this endpoint to TEA

Reply:
I think it's fine either way.